tools/pyclient/src/molgenis_emx2_pyclient/data_classes.py

```diff
@@ -0,0 +1,288 @@
+"""
+Classes for the data types Schema, Table and Column.
+"""
+from dataclasses import dataclass
+from typing import Literal
+
+from tools.pyclient.src.molgenis_emx2_pyclient.exceptions import NoSuchColumnException, NoSuchTableException
+
+
+@dataclass(eq=True, kw_only=True)
+class Column:
+    """Class for representing a column in the database."""
+    id: str
```

Comment:
the *id fields are autogenerated. What is your purpose with this API? If it is to manipulate the schema than I strongly recommend to remove the *id fields. Otherwise you will seed great confusion. If you however need to also generate graphql queries you need them. You might then consider to create separate data classes to make this distinction explicit